01-algorithms/02-algorithms-divide-conquer.md

```diff
@@ -46,246 +44,163 @@ QuickSort is an algorithm which takes a divide-and-conquer approach to sorting a
-_Fig. Example run of merge sort showing the index calculations used to split and merge the arrays. Note the convention used of the end index being exclusive._
-
-As we can see in the image above, in the first _divide_ step, the original array of size eight gets divided into two subarrays of size four each. This is done by setting the _starting index_ to _0_, the index of the first element in the array, and the _ending index_ is set to one past the last element in the array, using the convention of the ending index being exclusive. The _midway index_ is then computed using the formula:
+ _Fig. Example run of merge sort showing the index calculations used to split and merge the arrays. Note the convention used of the end index being exclusive._
 
-<div style="margin:auto;width:fit-content;">
+### Example: Binary Search (Decrease-and-Conquer)
 
-_midway index_ = ⌊(_starting index_ + _ending index_) / 2⌋
-
-</div>
+Binary search can be considered a decrease-and-conquer algorithm because it decreases the size of the array by half with each step. No matter how large the original array was, at each step we'll be able to discard half the remaining data from consideration! The half-sized array will then be analyzed with the same step as the full-sized array.
 
-Where ⌊⌋ denotes the mathematical _floor_ operation, or integer truncation.
+Binary search is technically not considered a divide-and-conquer algorithm but instead a decrease-and-conquer algorithm because divide-and-conquer algorithms typically have *two or more* subproblems which are generated from the main problem. In contrast, the binary search algorithm decreases to one sub-problem.
 
-### Divide
+Here's a description of the binary search algorithm:
 
-For the first _divide_ step, the _midway index_ will be _(0+8)/2_ = _4_.
+1. Starting with a sorted array, we check whether it contains a particular value by comparing the search value with the element in the middle of the array.
+1. If we find the value we return the position where it was found.
+1. If we don't find the value, we determine whether it would be in the left or right half of the array by seeing whether it is smaller or larger than the middle element. Remember, we can make this decision because _we assume the array is sorted!_ Then we perform a binary search on the selected half.
```

Comment:
```suggestion
1. If we don't find the value, we determine whether the value would be in the left or right half of the array by seeing whether the value is smaller or larger than the middle element. Remember, we can make this decision because _we assume the array is sorted!_ Then we perform a binary search on the selected half.
```